### DISCUSSION TITLE
Shopify Theme Lab 3 with Vue 3

### DISCUSSION THREAD

**Discussion by sergejcodes** [uicrooks/shopify-theme-lab#52] [Announcements]
> It's still safe to use the latest v2.x release. 

👉 [vue3 branch](https://github.com/uicrooks/shopify-theme-lab/tree/vue3)

While everything works, the official `Vue 3` tooling isn't 100% finished, so you might experience a slightly worse developer experience than with `Theme Lab` 2.x and `Vue` 2.x. Once the official `Vue` tooling is in better shape `Theme Lab 3` will be released.

**Comment by ScottJr** [comment]
@sergejcodes - Thank you so much for all your hard work!

**Reply by sergejcodes** [reply]
💙

**Comment by sergejcodes** [comment]
Shopify Theme Lab v3 is here 🎉 and it's using `Vue 3`. 
The latest Beta Vue Devtools plugin for Chrome now supports all features used by Theme Lab!
The vue3 branch was merged into master and deleted.
